package.json

```diff
@@ -89,6 +89,7 @@
     "rollup-plugin-url": "^3.0.0",
     "semantic-release": "^15.13.12",
     "styled-components": "^4.4.0",
+    "tippy.js": "^5.1.4",
```

Comment:
This is already in included in **Tippy** React wrapper ( `@tippy.js/react` ) which is already declared as part of our `"dependencies"`.

Why add it to `"devDependencies"` then? Well, in order to [import the correct version from the CDN](https://github.com/zopaUK/react-components/blob/924e401df85d26d58e3820c10faf35dab2bd39ad/src/components/styles/GlobalStyles.tsx#L33-L34) we need to somehow be able to read the current version of **Tippy JS** used, hence this "hack".

Reply:
Ahhh yez good point, let me try 👍

Reply:
I'm actually reading from node_modules, so thinking this back there's no need to add it to `devDependencies`: it will always be there because `@tippy.js/react` installs it.